lastcomment.py

```diff
@@ -140,8 +144,19 @@ def main():
                         help='only list hits for a specific project')
 
     args = parser.parse_args()
-    print_last_comments(args.name, int(args.count),
-                        args.message, args.project, args.votes)
+    names = [args.name]
+    if args.file:
+        with open(args.file) as f:
+            names = [l.rstrip() for l in f]
+
+    for n in names:
+        print 'Checking name: %s' , n
```

Comment:
This should be:  print 'Checking name: %s' % n

otherwise we get:
  Checking name: %s Joe Gordon